integreat_cms/cms/templates/linkcheck/links_by_filter.html

```diff
@@ -131,4 +131,42 @@ <h1 class="heading">
     </form>
     {% url 'linkcheck' as url %}
     {% include "pagination.html" with url=url chunk=page_obj %}
+    {% if request.user.expert_mode %}
+        <div class="flex-wrap relative w-auto mt-12">
+            <h3 class="font-bold text-xl">{% translate "Replace region link" %}</h3>
+            <div class="flex mt-2 mb-2">
+                <input id="replace_region_from"
+                       placeholder="{% translate "Old region slug" %}"
+                       class="pr-8 appearance-none block bg-gray text-gray-800 placeholder-gray-800 border border-gray-400 rounded py-3 px-4 leading-tight"/>
+                <i icon-name="arrow-right" class="inline-block ml-2"></i>
+                <input id="replace_region_to"
+                       placeholder="{% translate "New region slug" %}"
+                       class="pr-8 appearance-none block bg-white text-gray-800 placeholder-gray-800 border border-gray-400 rounded py-3 px-4 leading-tight ml-2"/>
+                <label id="region-replace-btn"
+                       class="inline-block cursor-pointer bg-blue-500 hover:bg-blue-600 text-white font-bold py-2 px-4 m-0 rounded ml-2"
+                       data-url="{% url 'replace_region_link' region_slug=request.region.slug %}">
+                    <i icon-name="link" class="inline-block pr-1"></i>
+                    {% translate "Replace links" %}
+                </label>
```

Comment:
Another tiny suggestion - to use the button tag for the "Replace links" button so that our common css style for buttons is applied

```suggestion
            <div class="flex mt-2 mb-2 gap-2">
                <input id="replace_region_from"
                       placeholder="{% translate "Old region slug" %}"
                       class="pr-8 appearance-none block bg-gray text-gray-800 placeholder-gray-800 border border-gray-400 rounded py-3 px-4 leading-tight"/>
                <i icon-name="arrow-right" class="inline-block"></i>
                <input id="replace_region_to"
                       placeholder="{% translate "New region slug" %}"
                       class="pr-8 appearance-none block bg-white text-gray-800 placeholder-gray-800 border border-gray-400 rounded py-3 px-4 leading-tight"/>
                <button id="region-replace-btn"
                        class="btn"
                        data-url="{% url 'replace_region_link' region_slug=request.region.slug %}">
                    <i icon-name="link"></i>
                    {% translate "Replace links" %}
                </button>
```